app/main/forms.py

```diff
@@ -73,6 +73,8 @@ def validate_email(self, email):
         :param email: The filtered email
         :return:
         """
+        if email.data and email.data .find('@') <= 0:
+            email.data += '@records.nyc.gov'
```

Comment:
Use the config variable `EMAIL_DOMAIN` instead of hardcoding here.

Reply:
Done